assets/src/hooks/useDetour.ts

```diff
@@ -59,11 +62,13 @@ export const useDetour = (routePatternId: RoutePatternId) => {
         (result) => {
           if (shouldUpdate) {
             setMissedStops(result?.missedStops || undefined)
+            setRouteSegments(result?.routeSegments || undefined)
```

Comment:
suggestion(non-blocking): what do you think about combining the `missedStops` and `routeSegments` states into a single `finishedDetour` state?

Reply:
@firestack how does [this](https://github.com/mbta/skate/pull/2455/commits/d5ff1dffbf0d540f1f725dc0c536720aa574044b) look? 

It goes halfway - it uses `finishedDetour` as internal state, but still exposes `missedStops` and `routeSegments` separately. I kind of prefer this, especially with the refactor that you and I were working on a few days ago, but I'm open to discussion about it.

Reply:
> but still exposes `missedStops` and `routeSegments` separately

That's exactly what I was intended/thinking. LGTM thanks!